content/tutorial-deep-learning-on-mnist.ipynb

```diff
@@ -0,0 +1,1017 @@
+{
```

Comment:
<p>Suggestion: add a note about why it's necessary to use the gray colormap</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Thanks for this suggestion. Here are the changes:

```diff
# reshape to (28, 28) to have a valid shape for displaying purposes.
...
- # Take the 60,000th image (index 59999) from the training set, reshape to (28x28) if necessary.
+ # Take the 60,000th image (index 59,999) from the training set,
+ # reshape from (784, ) to (28, 28) to have a valid shape for displaying purposes.
...
- # Set the color mapping to grayscale.
+ # Set the color mapping to grayscale to have a black background.
```